crypto/currency-exchange/README.md

```diff
@@ -0,0 +1,137 @@
+# Currency Exchange
+
+## Firestore Collections
+
+- Currency
+  > this will contain details about the accepted currencies at crypto-website
+
+## Currency Object
+
+```
+{
+    "id": <currency_id>,
+    "name": <currency_name>,
+    "quantity": <available_quantity>
+}
+```
+
+## Currency Exchange Object
+
+```
+{
+    from: {
+        "id": <currency_id>,
+        "name": <currency_name>
+    },
+    to: {
+        "id": <currency_id>,
+        "name": <currency_name>
+    },
+    value: <exchange_value>
+}
+```
+
+## Exchange Request Object
+
+```
+{
+  from: <src_currency_id>,
+  to: <trgt_currency_id>,
+  quantity: <selected_quantity>
+}
```

Comment:
How do we know which user is requesting this exchange? How do we know if it succeeded?